common/pref_names.cc

```diff
@@ -77,4 +77,5 @@ const char kPlayYTVideoInBrowserEnabled[] =
     "brave.play_yt_video_in_browser_enabled";
 const char kBackgroundVideoPlaybackEnabled[] =
     "brave.background_video_playback";
+const char kSafetynetCheckFailed[] = "safetynetcheck.failed";
```

Comment:
does this get used somewhere? I see code that writes it, but nothing that reads it

Reply:
Not yet, it will be, like we do in the old repo https://github.com/brave/browser-android-tabs/blob/base-77.0.3865.116/chrome/browser/ui/webui/chrome_web_ui_controller_factory.cc#L371. It is covered by that issue https://github.com/brave/brave-browser/issues/6567

Reply:
ok